uxarray/integrate.py

```diff
@@ -0,0 +1,49 @@
+"""xarray accessor functions for integration with uxarray."""
+import xarray as xr
+import uxarray as ux
+from .helpers import calculate_face_area
+
+
+@xr.register_dataset_accessor('integrate')
```

Comment:
This works but is not recommended. This probably overwrites the existing [Dataset.integrate](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.integrate.html). Instead the recommendation is to create a namespace using the accessor registration e.g. `@xr.register_dataset_accessor('ugrid')` and then do `ds.ugrid.integrate`. 

See the example [here](https://docs.xarray.dev/en/stable/internals/extending-xarray.html) where the namespace is `"geo"` and `plot` is a method in that namespace

Reply:
@dcherian , assuming such a namespace, "ugrid", is created, then a call to `integrate` would look like:

```
grid = ux.Grid("filename")
grid.ds.ugrid.integrate("psi")
```

which wouldn't look elegant. Is there a mechanism / best practice that Xarray recommends to the Xarray-extending packages to hide this chain to make the call look like:

'grid.integrate("psi")' 

instead? 

This is because, even though we want to extend Xarray as recommended, we would like the UXarray functionality to show up in the user API as its own, not through `Xarray.dataset`. I'd think of writing wrapper functions for such cases but it would clutter the codebase much.

Reply:
The xgcm-like syntax is 
```
grid = xgcm.Grid(...)
grid.integrate(ds.psi, axis=("X", "Y"))
```

But since you're making `ds` a property of `Grid` you can do
```
grid.integrate("psi")
```

This suggests `integrate` should be a method on the `Grid` class, which makes sense.